website/content/docs/ecs/terraform/secure-configuration.mdx

```diff
@@ -49,29 +74,43 @@ Use the [`acl-controller` terraform module](https://registry.terraform.io/module
+  version                           = "<version>"
+
   consul_bootstrap_token_secret_arn = aws_secretsmanager_secret.bootstrap_token.arn
   consul_server_http_addr           = "https://consul-server.example.com:8501"
   consul_server_ca_cert_arn         = aws_secretsmanager_secret.ca_cert.arn
   ecs_cluster_arn                   = "arn:aws:ecs:my-region:111111111111:cluster/consul-ecs"
-  region                            = "my-region"
+  region                            = "us-east-1"
   subnets                           = ["subnet-abcdef123456789"]
   name_prefix                       = "consul-ecs"
 }
 ```
 
-The `name_prefix` parameter is used to prefix any secrets that the ACL controller will
-update in AWS Secrets Manager.
+The following table explains the `acl-controller` input variables:
 
--> **NOTE:** Make sure that the `name_prefix` is unique for each ECS cluster where you are
-deploying this controller.
+| Input Variable                      | Type     | Description                                                                                                                                                                                                                                |
+| ----------------------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `source`                            | string   | The location of the `acl-controller` module source.                                                                                                                                                                                        |
+| `version`                           | string   | The version of the `acl-controller` module.                                                                                                                                                                                                |
+| `consul_bootstrap_token_secret_arn` | string   | The Secrets Manager secret containing an ACL token for the ACL controller. This must contain `acl:write` and `operator:write` permissions.                                                                                                 |
+| `consul_server_http_addr`           | string   | The HTTP(S) address of the Consul servers.                                                                                                                                                                                                 |
+| `consul_server_ca_cert_arn`         | string   | (optional) The Secrets Manager secret containing the CA cert for HTTPS communication with Consul servers. Required if the server's certificate is self-signed or signed by an internal CA. This is not required for Consul servers in HCP. |
```

Comment:
```suggestion
| `consul_server_ca_cert_arn`         | string   | (Optional) The Secrets Manager secret containing the CA cert for HTTPS communication with Consul servers. Required if the server's certificate is self-signed or signed by an internal CA. This is not required for Consul servers in HCP. |
```
This is the only field described as optional. Are the others required?

Reply:
Yep. The other inputs are required. I updated the sentence prior to this table to indicate that, but still call this one out as optional since it important but not always required. Let me know what you think.